src/diseaseProg.jl

```diff
@@ -492,12 +492,12 @@ end
 
 # TODO - MANUAL! - this function is VERY specific to current health state setup, and needs to be manually edited if number of health states change
 function inpFunc_testSpecifications(
-	PCR_FNR_I1_to_R2 = [ 0.9,  0.4, 0.15, 0.35, 0.5, 0.8],
-    PCR_FPR = 0.01,
-    antigen_FNR_I1_to_R2 = [ 0.95, 0.6, 0.35, 0.45, 0.6, 0.9],
-    antigen_FPR = 0.1,
-    antibody_FNR_I1_to_R2 = [0.99, 0.85, 0.8, 0.65, 0.3, 0.05],
-    antibody_FPR_S_to_I4 = [0.05, 0.04, 0.03, 0.02, 0.01];
+	PCR_FNR_I1_to_R2::Array = [ 0.9,  0.4, 0.15, 0.35, 0.5, 0.8],
+    PCR_FPR::Float64 = 0.01,
+    antigen_FNR_I1_to_R2::Array = [ 0.95, 0.6, 0.35, 0.45, 0.6, 0.9],
+    antigen_FPR::Float64 = 0.1,
+    antibody_FNR_I1_to_R2::Array = [0.99, 0.85, 0.8, 0.65, 0.3, 0.05],
+    antibody_FPR_S_to_I4::Array = [0.05, 0.04, 0.03, 0.02, 0.01];
```

Comment:
We do not need `kwargs...`, check the python version once.

Reply:
Sure, will remove it